productsapp/components/ProductsList.js

```diff
@@ -1,79 +1,84 @@
 import * as React from 'react';
 import { Text, View, Image, TouchableOpacity } from 'react-native';
-import Product from '/Product'
+import Product from '/Product';
 import { Font } from 'expo';
 import ProductView from './ProductView';
 import styles from './Styles';
 
 export default class ProductsList extends React.Component<{}> {
-   componentDidMount() {
+  constructor(props) {
+    super(props);
+    this.state = { showProduct: false };
+  }
+
+  componentDidMount() {
     Font.loadAsync({
-      'vincHand': require('../assets/fonts/VINCHAND.ttf'),
+      vincHand: require('../assets/fonts/VINCHAND.ttf'),
```

Comment:
https://docs.expo.io/versions/latest/guides/using-custom-fonts
need to implement **Waiting for the font to load before rendering** part of tutorial

![capture](https://user-images.githubusercontent.com/24648180/49930517-d338bc00-fed5-11e8-87d3-5558dc4f916e.PNG)

Reply:
I think your environment have this font and you don't catch error in this place